firefox_puppeteer/api/prefs.py

```diff
@@ -202,3 +202,13 @@ def set_pref(self, pref_name, value):
             """, script_args=[pref_name, value])
 
         assert retval
+
+    def persist_prefs(self):
+        """Persist current pref values to the default prefs file (typically prefs.js)
+        """
+        with self.marionette.using_context('chrome'):
+            self.marionette.execute_script("""
+              Cu = Components.utils;
+              Cu.import("resource://gre/modules/Services.jsm");
+              Services.prefs.savePrefFile(null);
```

Comment:
OMG!!! This is https://github.com/mozilla/firefox-ui-tests/blob/master/firefox_ui_harness/testcase.py#L35.

So we have two options:

1) Each test takes care of removing the prefs as set, which we most likely wanna do. I actually never thought about this problem.
2) We define an exclusion flag for tearDown.